frontend/packages/console-app/src/components/quick-starts/utils/quick-start-context.ts

```diff
@@ -124,12 +134,19 @@ export const useValuesForQuickStartContext = (): QuickStartContextValues => {
         let updatedTaskStatus;
 
         if (status === QuickStartStatus.NOT_STARTED) {
+          fireTelemetryEvent('Quick Start Initiated', {
+            id: activeQuickStartID,
+            type: 'start',
```

Comment:
Add quick start `name`

Reply:
After investigation, the adding the name at this point would require some api refactoring and not worth the effort for the initial telemetry release.